CHANGES.md

```diff
@@ -75,6 +83,8 @@ Released on December 3, 2022.
  -  Added `IBlockPolicy.GetMinBlockProtocolVersion()` interface method.
     [[#2593]]
 
+### Behavioral changes
+
```

Comment:
I think these lines may have been added by mistake?

Reply:
Yes, you're right. I removed these lines.